### DISCUSSION TITLE
为什么插件import直接报错了

### DISCUSSION THREAD

**Discussion by vernvn** [tinymce-plugin/tinymce-plugin.github.io#12] [Show and tell (演示和告知)]
![image](https://user-images.githubusercontent.com/17755520/199633931-ff7f46f2-ecf9-40e0-9e15-bc9e3a66444d.png)

**Comment by Five-great** [comment]
@vernvn 你需要如下配置 详情见[tinymce-plugin/tpImportword 快速上手](https://tinymce-plugin.gitee.io/packages/tp-importword/quickStart.html)
![image](https://user-images.githubusercontent.com/43601963/199634471-eee7c97e-2867-4064-ab3a-1ef9b7de1c4f.png)

**Reply by vernvn** [reply]
是这么配置的，报错了，可能是版本得问题把

**Reply by vernvn** [reply]
项目比较老，vue2.6+版本的，tinymce5.1.0 & @tinymce/tinymce-vue3.0.1

**Reply by Five-great** [reply]
有类型检测的问题，你可以参试 将tinymce-plugin 包本地化 放到 static 或者public 文件夹下 通过配置 `external_plugins`
例如：
我将 tinymce-plugin 包中 tpImportword文件夹 移动到 public/tinymce/plugins 目录下
进行如下配置
![image](https://user-images.githubusercontent.com/43601963/199639313-bee77261-75da-4054-96d9-cd718d2599fc.png)

![AU%~Y{@8@_ORCZA$5S`)3`E](https://user-images.githubusercontent.com/43601963/199639291-6b21cf96-f6ef-4255-9bb9-2f32a5e4a849.png)

**Reply by vernvn** [reply]
好的，我试下，感谢

**Reply by vernvn** [reply]
可以了，非常感谢您！再请教您2个问题：1、有没有api或者插件可以支持将富文本输入的内容导出为word文件；2、是否有api或者配置项可以支持直接读取word里面得内容而不是通过toolbar先导入

**Reply by Five-great** [reply]
1. 官方有 好像 导出pdf word 你可以去英文官网找找[官方文档](https://www.tiny.cloud/docs/)
2. 可以调用 api 自定义 不通过toolbar 详情见[配置参考](https://tinymce-plugin.gitee.io/packages/tp-importword/configure.html)
![image](https://user-images.githubusercontent.com/43601963/199642819-ab4f190d-bb1a-4d4b-a212-6bf9011af6ac.png)

**Reply by vernvn** [reply]
非常感谢您耐心得回复

**Reply by vernvn** [reply]
配置参考里得api还是会有一个选择word文件得动作过程，能不能默认直接就解析word内容呢

**Reply by vernvn** [reply]
为什么有的word导入得时候，提示导入成功，但是没有内容显示在富文本里啊，word里面是有内容得
![image](https://user-images.githubusercontent.com/17755520/199678973-75170e2b-dd2b-4351-aa40-03f4e5de7b29.png)
![image](https://user-images.githubusercontent.com/17755520/199679076-e29d2b8b-6721-4552-93a3-9e94acea881d.png)

**Reply by Five-great** [reply]
你是vue v-model 么 你可以加我QQ 985540932  空了帮你看看
